### PR TITLE
Added CONTRIBUTORS.md file

### LINKED ISSUE #33: Create CONTRIBUTORS.md
This will be used as a way to keep track of who contributed to this project.

This should be a table format and it should allow a contributor to send a PR adding their GitHub name, Discord name, as well as anything else they want to add.

### PR DESCRIPTION
**Pull request by mehta-avi**
## Issue
Closes #33

## Changes proposed 
Added a table which lists the contributor's Github username, Discord username, and any other additional info they would like to share

## Screenshots
image

### CODE REVIEW

**Comment by ykdojo** [comment]
Thank you. Let's add an additional column for Contributions.

Also, to be consistent with sentence casing in other places, I think we should use `GitHub username` and `Discord username`. Let me know what you think

**Comment by mehta-avi** [comment]
Yeah, I think those are fair amendments that we can add. Pushed again with those changes reflected!

**Comment by ykdojo** [comment]
Thank you. Created #40 for this.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Contributors
+
+| GitHub username | Discord username | Contributions | Additional Info |
```
Comment (ykdojo) [inline CONTRIBUTORS.md:3]:
```suggestion
| GitHub username | Discord username | Contributions | Additional info |
```

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+|-----------------|------------------|---------------|-----------------|
+
+| [mehta-avi](https://github.com/mehta-avi) | rbwings | Documentation |[My website](avimehta.info) |
```
Comment (ykdojo) [inline CONTRIBUTORS.md:6]: It doesn't seem to be rendering as a table.

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.md (hunk 2 above)
Comment (mehta-avi) [reply CONTRIBUTORS.md]: Fixed this. There was an additional line break which I removed.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (mehta-avi) [reply CONTRIBUTORS.md]: Done.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Contributors
+
+| GitHub username | Discord username | Contributions | Additional info |
+|-----------------|------------------|---------------|-----------------|
+| [mehta-avi](https://github.com/mehta-avi) | rbwings | Documentation |[My website](avimehta.info) |
```
Comment (ykdojo) [inline CONTRIBUTORS.md:5]: I think the link needs to be full for it to work on GitHub

Comment (ykdojo) [inline CONTRIBUTORS.md:5]: Also contributions should have more details

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.md (hunk 3 above)
Comment (mehta-avi) [reply CONTRIBUTORS.md]: Understood. Went back and added details on what I have been involved with so far

**Review by ykdojo — APPROVED** [review]

**Pull request merged**